cryptus-app/components/wallet_viewer/nft_details/nft_infos/nft_info.tsx

```diff
@@ -0,0 +1,42 @@
+//react and css
+import React, { useState, useEffect } from "react";
+import NFTBuyButton from "./nft_buy_button/nft_buy_button";
+import s from "./nft_info.module.scss";
+
+//external exports
+
+//internal imports
+import { nft } from "../../../../lib/data_types";
+
+export default function NFTInfo(props: { nft: nft }) {
+
+    return (
+        <div className={s.container}>
+            <div className={s.left}>
+                <NFTCollection {...props} />
+                <NFTName {...props} />
+            </div>
+            <NFTBuyButton {...props} />
+        </div>
+    );
+}
+
+const NFTCollection = (props: { nft: nft }) => {
```

Comment:
NftCollection. See here, you could argue a function was a good thing, but then again it's only a one-liner